MUON/MUONsim/AliMUONTriggerGeometryBuilder.cxx

```diff
@@ -1623,7 +1623,7 @@ void AliMUONTriggerGeometryBuilder::BuildChamberTypeE(Int_t& iVolNum, Int_t icou
     Float_t zRatio = AliMUONConstants::DefaultRatioTriggerChamber(icount);      
     Float_t xEnv = (fgkDXZERO+fgkXMAX/2.)*zRatio;
    
-   Double_t dpar[3];    
+   Double_t dpar[3] = {0., 0., 0.};     
```

Comment:
Looking at the code, it seems that the RPC type D, E, and F are the RPCs of type "long". ~
For some reason they where built separately, but one could have simply built the base shape and then properly place them in space. Indeed, AFAIK, the type D and F only differ by the y position, while the type E differ from the other also for Z position.
Anyways, in type D and F the dpar is initialised to the values:
dpar[0] = (fgkXMAX/2.)*zRatio;
dpar[1] =  fgkYMIN*zRatio;
dpar[2] = 0.4;
I have to say that I do not fully understand the meaning of those values, but I guess that the type E should have the same initialisation.